llvm/test/Verifier/llvm.loop.estimated_trip_count.ll

```diff
@@ -0,0 +1,61 @@
+; Test "llvm.loop.estimated_trip_count" validation
+
+; DEFINE: %{RUN} = opt -passes=verify %t -disable-output 2>&1 | \
+; DEFINE:   FileCheck %s -allow-empty -check-prefix
```

Comment:
The idiomatic way to do this is split-file. Tests that don't actually contain the tested code are inconvenient.